src/main/kotlin/gdx/liftoff/data/project/Project.kt

```diff
@@ -144,6 +144,7 @@ class Project(
   private fun saveProperties() {
     // Adding libGDX version property:
     properties["gdxVersion"] = advanced.gdxVersion
+    properties["gameVersion"] = advanced.version
```

Comment:
I think `projectVersion` is maybe a little more general, but still suggests that it is this project and not some dependency's version. I'll get the workflow running, check that, and merge if everything's OK. I can change the name of the property at any point before the next release, so there's no rush.